src/Features/CSharpTest/Microsoft.CodeAnalysis.CSharp.Features.UnitTests.csproj

```diff
@@ -13,8 +13,30 @@
     <ProjectReference Include="..\DiagnosticsTestUtilities\Microsoft.CodeAnalysis.Features.DiagnosticsTests.Utilities.csproj" />
     <ProjectReference Include="..\TestUtilities\Microsoft.CodeAnalysis.Features.Test.Utilities.csproj" />
   </ItemGroup>
+  <ItemGroup>
+    <!--
+      Not using ProjectReference since it does not execute the specified targets when building in VS.
+      See https://github.com/microsoft/msbuild/issues/5434.
+    -->
+
+    <CopyPublishedOutputProjectReference Include="..\..\Tools\SemanticSearch\ReferenceAssemblies\SemanticSearch.ReferenceAssemblies.csproj">
```

Comment:
curuiys why C# feature tests need a reference to this.

Reply:
So that we can test compilation of the queries. We need to create metadata references for the query compilation.